src/kirby/components/calendar/calendar.component.ts

```diff
@@ -0,0 +1,317 @@
+import 'reflect-metadata';
+import {
+  Component,
+  ViewChild,
+  ElementRef,
+  Input,
+  Output,
+  EventEmitter,
+  OnInit,
+  AfterViewInit,
+  LOCALE_ID,
+  Inject,
+  NgZone,
+} from '@angular/core';
+import * as moment from 'moment';
+
+import { CalendarHelper } from './helpers/calendar.helper';
+import { CalendarOptions } from './helpers/calendar-options.model';
+import { CalendarCell } from './helpers/calendar-cell.model';
+
+type CalendarDay = {
+  isCurrentMonth: boolean;
+  isToday: boolean;
+  isWeekend: boolean;
+  isPast: boolean;
+  isFuture: boolean;
+  isDisabled: boolean;
+};
+
+@Component({
+  selector: 'kirby-calendar',
+  templateUrl: './calendar.component.html',
+  styleUrls: ['./calendar.component.scss'],
+  providers: [CalendarHelper],
+})
+export class CalendarComponent implements OnInit, AfterViewInit {
+  @ViewChild('calendarContainer') calendarContainer: ElementRef;
+  @Output() dateChange = new EventEmitter<Date>();
+  @Input() disableWeekends = false;
+  @Input() disablePastDates = false;
+  @Input() disableFutureDates = false;
```

Comment:
disableFutureDates is missing from sample HTML as well as properties described in Cookbook!

Reply:
Fixed